SPECS/kernel-headers/kernel-headers.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Linux API header files
 Name:           kernel-headers
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment